server/src/test/java/org/elasticsearch/action/admin/indices/rollover/MetaDataRolloverServiceTests.java

```diff
@@ -212,7 +212,7 @@ public void testValidation() {
         assertThat(exception.getMessage(), equalTo("source alias [" + aliasWithNoWriteIndex + "] does not point to a write index"));
         exception = expectThrows(IllegalArgumentException.class, () ->
             MetaDataRolloverService.validate(metaData, randomFrom(index1, index2)));
-        assertThat(exception.getMessage(), equalTo("source alias is a concrete index"));
+        assertThat(exception.getMessage(), equalTo("source alias is not an alias"));
```

Comment:
I think the change to the exception message is not really a big issue. But with data streams also appearing, it could be nice to stick to the old message since that would give you some information on whether it is a datastream or concrete index. I think we could easily generate such a message by adding a display name ("concrete index") to the IndexAbstract.Type enum?

Also, this does make me wonder if `Type.INDEX` should be `Type.CONCRETE_INDEX` for consistency with that display name? Not truly important to me, feel free to ignore this part.